Makefile

```diff
@@ -62,6 +62,27 @@ GOASSETSBUILDER := $(GOBUILDDIR)/bin/go-assets-builder$(shell go env GOEXE)
 
 BUILDTIME = $(shell go run "$(ROOT)/tools/dategen/")
 
+GOBUILDLDFLAGS := -X $(REPOPATH)/pkg/version.version=$(VERSION) -X $(REPOPATH)/pkg/version.buildDate=$(BUILDTIME) -X $(REPOPATH)/pkg/version.build=$(COMMIT)
+GOBUILDGCFLAGS :=
+
+# Go Strip Section
+GOBUILDSTRIP ?= 1
+ifeq ($(GOBUILDSTRIP),1)
+GOBUILDLDFLAGS += -w -s
+endif
+
+# Go Disable function inlining
+GOBUILDDISABLEFUNCTIONINLINING ?= 1
```

Comment:
How much it affect the operator performance though? Wondering to see the benchmarking results - I think we have some benchmarking tests there.